alf/examples/ddpg_pendulum.gin

```diff
@@ -8,9 +8,41 @@ NUM_PARALLEL_ENVIRONMENTS=1
 create_environment.env_name="Pendulum-v0"
 create_environment.num_parallel_environments=%NUM_PARALLEL_ENVIRONMENTS
 
+
+actor/ActorNetwork.input_tensor_spec=%observation_spec
+actor/ActorNetwork.action_spec=%action_spec
+actor/ActorNetwork.fc_layer_params=(100,100)
+actor/Adam.lr=1e-4
+
+critic/CriticNetwork.input_tensor_spec=(%observation_spec, %action_spec)
+critic/CriticNetwork.joint_fc_layer_params=(100,100)
+critic/Adam.lr=1e-3
+
+# uncomment to use parallel critics
+#DdpgAlgorithm.use_parallel_network=1
+#DdpgAlgorithm.num_replicas=2
+DdpgAlgorithm.actor_network=@actor/ActorNetwork()
+DdpgAlgorithm.critic_network=@critic/CriticNetwork()
+DdpgAlgorithm.actor_optimizer=@actor/Adam()
+DdpgAlgorithm.critic_optimizer=@critic/Adam()
+
+DdpgAlgorithm.target_update_period=5
+OneStepTDLoss.td_error_loss_fn=@losses.element_wise_huber_loss
+
+
+# training config
+TrainerConfig.initial_collect_steps=1000
```

Comment:
some of this can be put into ddpg.gin: whole_replay_buffer_training, clear_replay_buffer

Reply:
Changed according to comments.